docs/validators/validator-faq.md

```diff
@@ -64,19 +65,48 @@ Out of all the candidates that signaled themselves, the 100 with the most stake
 * **Year 9:** 300
 * **Year 10:** 300
 
+## Testnet
+
 ### How can I join the testnet?
 
 The Testnet is a great environment to test your validator setup before launch.
 
-We view testnet participation as a great way to signal to the community that you are ready and able to operate a validator. You can find all relevant information about the [testnet and more here](/getting-started/full-node.md).
+We view testnet participation as a great way to signal to the community that you are ready and able to operate a validator. You can find all relevant information about the testnet [here](https://github.com/cosmos/cosmos-sdk/tree/develop/cmd/gaia/testnets) and [here](https://github.com/cosmos/testnets).
+
+### What are the different types of keys?
+
+In short, there are two types of keys:
+
+- **Tendermint Key**: This is a unique key used to sign block hashes. It is associated with a public key `cosmosvalpub`.
+    + Generated when the node is created with gaiad init.
+    + Get this value with gaiad tendermint show_validator
+    +M e.g. cosmosvalpub1zcjduc3qcyj09qc03elte23zwshdx92jm6ce88fgc90rtqhjx8v0608qh5ssp0w94c 
+
+- **Application keys**: These keys are created from the application and used to sign transactions. As a validator, you will probably use one key to sign staking-related transactions, and another key to sign governance-related transactions. Application keys are associated with a public key `cosmosaccpub` and an address `cosmosaccaddr`. Both are derived from account keys generated by `gaiacli keys add`.
```

Comment:
Clarify that this will own the validator

Reply:
`As a validator, you will probably use one key to sign staking-related transactions, and another key to sign governance-related transactions.` this is the first i've heard about this. is there a reason / justification / explanation somewhere?